pages/docs/tutorials/app-example-config.mdx

```diff
@@ -0,0 +1,672 @@
+
+We'll change a few values in the config file (`config/destinations/mailchimp.js`). We want to set:
+
+- Group ID for the Group of Profiles to export.
+- Mailchimp's list ID
+- The fields to include in the Mailchimp contact.
+- Any tags (as Grouparoo Groups) to attach to the Mailchimp contact.
+
+```js
+exports.default = async function buildConfig() {
+  return [
+    {
+      id: "newsletter",
+      name: "newsletter",
+      class: "destination",
+      type: "mailchimp-export",
+      appId: "mailchimp",
+      // Send everyone with a valid email address.
+      groupId: "all_emails",
+      options: {
+        // The list ID we set in .env.
+        listId: process.env.MAILCHIMP_LIST_ID,
+      },
+      // The properties to send to Mailchimp with the Profile. Mailchimp has a
+      // few predetermined fields, which is why `FNAME` and `LNAME` have a
+      // different structure than the others.
+      mapping: {
+        email_address: "email",
+        FNAME: "first_name",
+        LNAME: "last_name",
+        language: "language",
+      },
+      // This tells Grouparoo to tag Profiles in the
+      // `high_value_spanish_speakers` group with "High Value Spanish Speaker"
+      // in Mailchimp.
+      destinationGroupMemberships: {
+        "High Value Spanish Speaker": "high_value_spanish_speakers",
+      },
+    },
+  ];
+};
+```
+
+### Validate and Apply Config Changes
+
+You should now be all set to validate:
+
+    $ grouparoo validate
+
+    🦘 Grouparoo: validate
+
+    Validating 15 objects...
+    info: [ config ] validated App `mailchimp` (mailchimp)
+    ...
+    info: [ config ] validated Destination `newsletter` (newsletter)
+    ...
+
+    ✅ Validation succeeded - 15 config objects OK!
+
+And apply:
+
+    $ grouparoo apply
+
+    🦘 Grouparoo: apply
+
+    Applying 15 objects...
+    info: [ config ] updated App `mailchimp` (mailchimp)
+    ...
+    info: [ config ] updated Group `high_value_spanish_speakers` (high_value_spanish_speakers)
+    notice: [ run ] starting run run_fb4eba09-20cc-40f7-a462-93a0b66a72bf for group all_emails (all_emails)
+    ...
+
+    ✅ Config applied - 15 config objects up-to-date!
+
+## Step 9: Run
+
+Now you're all configured. Yay! 🎉
+
+We are now at the point where we have _essentially_ everything included in [the app-example-config project](https://github.com/grouparoo/app-example-config). The only thing we're missing is writing any tests to add confidence to the data you're exporting. More on that below.
+
+You're ready to run Grouparoo! Do it!
+
+    $ grouparoo start
+
+This will begin the process of importing data from SQLite into Grouparoo and exporting it to Mailchimp.
+
+After you run the command the server will go wild, and that's a good thing. But it's tough to follow. You can open your browser and run the web app for some visibility and observability into what's happening under the hood. The app is available at [localhost:3000](http://localhost:3000/).
```

Comment:
I *think* google dings you for linking to localhost/a URL that doesn't resolve.  May not a link here.